src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -18,28 +18,28 @@ public class DeleteCommand extends Command {
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
+            + ": Deletes the person identified by the index number in the displayed task list or the person's name.\n"
```

Comment:
i think this sentence could be clearer as im not quite sure what it means

Reply:
Oh I think this was a typo, thanks